Data-Structures/Stack.ts

```diff
@@ -1,4 +1,4 @@
-/* Stack data-structure. It's work is based on the LIFO method (last-IN-first-OUT).
+/* Stack data-structure. Its work is based on the LIFO method (last-IN-first-OUT).
```

Comment:
Unrelated change

Reply:
Oh yeah sorry, I forgot to mention that but I felt it was just an extremely small grammatical change. Do I need to make a separate PR for that? (It is just an apostrophe which is to be removed :sweat_smile: )

Reply:
I think we should keep it here as it's a very small change, IMO.

Reply:
These sort of changes mess up the pr diff, which is the reason a different pr would be better.

Reply:
> Oh yeah sorry, I forgot to mention that but I felt it was just an extremely small grammatical change. Do I need to make a separate PR for that? (It is just an apostrophe which is to be removed sweat_smile )

Accumulate fixes for small typos, formatting etc. on a separate branch, then eventually open a PR to fix all of them.

Reply:
Alright then :+1: